apps/examples/testcase/ta_tc/systemio/itc/itc_uart.c

```diff
@@ -373,15 +405,17 @@ static void itc_systemio_iotbus_uart_write_read_flush_p_multi_handle(void)
 	ret = iotbus_uart_write(h_uart1, sz_input_text1, sizeof(sz_input_text1));
 	TC_ASSERT_EQ_CLEANUP("iotbus_uart_write", ret < 0, false, iotbus_uart_stop(h_uart1); iotbus_uart_stop(h_uart2));
 
-	ret = iotbus_uart_write(h_uart2, sz_input_text2, sizeof(sz_input_text2));
-	TC_ASSERT_EQ_CLEANUP("iotbus_uart_write", ret < 0, false, iotbus_uart_stop(h_uart1); iotbus_uart_stop(h_uart2));
-
 	usleep(MICROSECOND);
 
 	ret = iotbus_uart_read(h_uart1, sz_output_text, sizeof(sz_output_text));
 	TC_ASSERT_EQ_CLEANUP("iotbus_uart_read", ret < 0, false, iotbus_uart_stop(h_uart1); iotbus_uart_stop(h_uart2));
 	TC_ASSERT_EQ_CLEANUP("iotbus_uart_read", strcmp(sz_input_text1, sz_output_text), 0, iotbus_uart_stop(h_uart1); iotbus_uart_stop(h_uart2));
 
+	ret = iotbus_uart_write(h_uart2, sz_input_text2, sizeof(sz_input_text2));
+	TC_ASSERT_EQ_CLEANUP("iotbus_uart_write", ret < 0, false, iotbus_uart_stop(h_uart1); iotbus_uart_stop(h_uart2));
```

Comment:
We usually use `TC_ASSERT_LEQ_CLEANUP` or `TC_ASSERT_GEQ_CLEANUP` for this case.
How about changing?

Reply:
I think we should leave it, as it is. because in the whole file they have used like this and if we change for one line then it won't look good and it does not affect the result so we can keep it same.
let me know still what you think?

Reply:
Ok fine. Let's keep it on as it is.